src/modules/ekf2/EKF/mag_control.cpp

```diff
@@ -399,49 +296,27 @@ void Ekf::checkMagBiasObservability()
 	_time_yaw_started = _time_delayed_us;
 }
 
-bool Ekf::canUse3DMagFusion() const
+bool Ekf::magFieldStrengthDisturbed(const Vector3f &mag_sample) const
```

Comment:
that's a merge issue, this function is replaced by `checkMagField`

Reply:
This particular case was laziness. The original idea as a condition for starting full mag_3d was that the bias corrected sample would need to fully pass the mag field checks. I'll drop it for now and can revisit it later with checkMagField if there's any merit.